crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
+    unreferenced_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
+
+      unless disk =~ /^\/dev\/disk\//
+        unreferenced_disks.push(disk) if node[:block_device][device_name].nil?
+        next
+      end
+
+      if path == "cciss"
```

Comment:
I don't think so because disk =~ /^\/dev\/disk\// strictly checks for only two parts like /dev/cciss but if there is something more to it  which I think will be the case like /dev/cciss/c0c0p1 it will not be checked above. but again there is not harm in moving it up.